checkuser.php

```diff
@@ -9,35 +9,71 @@
 include 'db.php';
 include 'lib/utility.php';
 
-$email  = htmlentities(trim($_POST['email']));
+$loginname  = htmlentities(trim($_POST['email']));
```

Comment:
Does htmlentities or some other method protect against sql injection attacks? For example, login name is entered as "guest@some.com;drop table user;" this would be flagged and an error thrown before it is concatenated to the sql and executed.

Reply:
> Does htmlentities or some other method protect against sql injection attacks? For example, login name is entered as "[guest@some.com](mailto:guest@some.com);drop table user;" this would be flagged and an error thrown before it is concatenated to the sql and executed.

Doubt it, https://www.php.net/manual/en/function.htmlentities.php
The original php code is legacy, written by others, and likely needs a complete security review. 
We could start ad-hoc adding bits of checking when we touch code, but I think a thorough review and a consistent coding pattern should be determined and applied globally.
Another issue should be created.

Reply:
Created https://github.com/ehb54/ultrascan-tickets/issues/304